### ISSUE TITLE
Error: "Class 'PiggyCustomEnchants\CustomEnchants\CustomEnchants' not found" (EXCEPTION)

### ISSUE THREAD

**Issue by AL3XPI** [Infernus101/KitUI#118]
## KitUI Issue
- [x] This issue isn't duplicated - you can check if it is by using the search bar located at the top left hand corner and select "Issues" on the left.
- [x] This issue includes appropriate markdown for sections - e.g. code blocks for crash dumps.
- [x] This issue is understandable in English.

## Issue description
Server crashes when you click on a kit with an enchanted sword (unbreaking 3). 1 of my 2 kits (1 has unenchanted stone sword other has enchanted diamond sword).

## Reproducing the issue
1. ...
2. ...

## Client information
PocketMine-MP Version: (insert commit hash here) 3.9.2
Kit-UI Version: (insert commit hash or version from plugin.yml here) 1.3.5
MCPE Version: (insert game version here) 1.12.0
Plugin Version: (insert commit hash here)
PHP version: 7.2

## Optional information
...

```
[19:01:46] [Server thread/CRITICAL]: Error: "Class 'PiggyCustomEnchants\CustomEnchants\CustomEnchants' not found" (EXCEPTION) in "plugins/KitUI.phar/src/Infernus101/KitUI/Kit" at line 159
[19:01:46] [Server thread/DEBUG]: #0 plugins/KitUI.phar/src/Infernus101/KitUI/Kit(110): Infernus101\KitUI\Kit->loadItem(integer 276, integer 0, integer 1, string[7] DEFAULT, string[2] 17, string[1] 3)
[19:01:46] [Server thread/DEBUG]: #1 plugins/KitUI.phar/src/Infernus101/KitUI/Kit(71): Infernus101\KitUI\Kit->add(object pocketmine\Player)
[19:01:46] [Server thread/DEBUG]: #2 plugins/KitUI.phar/src/Infernus101/KitUI/UI/windows/KitInfo(94): Infernus101\KitUI\Kit->equipKit(object pocketmine\Player)
[19:01:46] [Server thread/DEBUG]: #3 plugins/KitUI.phar/src/Infernus101/KitUI/UI/windows/KitInfo(36): Infernus101\KitUI\UI\windows\KitInfo->select(string[5] true.)
[19:01:46] [Server thread/DEBUG]: #4 plugins/KitUI.phar/src/Infernus101/KitUI/PlayerEvents(80): Infernus101\KitUI\UI\windows\KitInfo->handle(object pocketmine\network\mcpe\protocol\ModalFormResponsePacket)
[19:01:46] [Server thread/DEBUG]: #5 src/pocketmine/plugin/MethodEventExecutor(38): Infernus101\KitUI\PlayerEvents->onDataPacket(object pocketmine\event\server\DataPacketReceiveEvent)
[19:01:46] [Server thread/DEBUG]: #6 src/pocketmine/plugin/RegisteredListener(98): pocketmine\plugin\MethodEventExecutor->execute(object Infernus101\KitUI\PlayerEvents, object pocketmine\event\server\DataPacketReceiveEvent)
[19:01:46] [Server thread/DEBUG]: #7 src/pocketmine/event/Event(99): pocketmine\plugin\RegisteredListener->callEvent(object pocketmine\event\server\DataPacketReceiveEvent)
[19:01:46] [Server thread/DEBUG]: #8 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\event\Event->call()
[19:01:46] [Server thread/DEBUG]: #9 src/pocketmine/network/mcpe/protocol/BatchPacket(123): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(object pocketmine\network\mcpe\protocol\ModalFormResponsePacket)
[19:01:46] [Server thread/DEBUG]: #10 src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(110): pocketmine\network\mcpe\protocol\BatchPacket->handle(object pocketmine\network\mcpe\PlayerNetworkSessionAdapter)
[19:01:46] [Server thread/DEBUG]: #11 src/pocketmine/Player(3126): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(object pocketmine\network\mcpe\protocol\BatchPacket)
[19:01:46] [Server thread/DEBUG]: #12 src/pocketmine/network/mcpe/RakLibInterface(169): pocketmine\Player->handleDataPacket(object pocketmine\network\mcpe\protocol\BatchPacket)
[19:01:46] [Server thread/DEBUG]: #13 vendor/pocketmine/raklib/src/server/ServerHandler(99): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string[18] 192.168.1.99 51046, object raklib\protocol\EncapsulatedPacket, integer 0)
[19:01:46] [Server thread/DEBUG]: #14 src/pocketmine/network/mcpe/RakLibInterface(109): raklib\server\ServerHandler->handlePacket()
[19:01:46] [Server thread/DEBUG]: #15 src/pocketmine/network/mcpe/RakLibInterface(99): pocketmine\network\mcpe\RakLibInterface->process()
[19:01:46] [Server thread/DEBUG]: #16 vendor/pocketmine/snooze/src/SleeperHandler(123): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
[19:01:46] [Server thread/DEBUG]: #17 vendor/pocketmine/snooze/src/SleeperHandler(85): pocketmine\snooze\SleeperHandler->processNotifications()
[19:01:46] [Server thread/DEBUG]: #18 src/pocketmine/Server(2348): pocketmine\snooze\SleeperHandler->sleepUntil(double 1563840106.9726)
[19:01:46] [Server thread/DEBUG]: #19 src/pocketmine/Server(2205): pocketmine\Server->tickProcessor()
[19:01:46] [Server thread/DEBUG]: #20 src/pocketmine/Server(1784): pocketmine\Server->start()
[19:01:46] [Server thread/DEBUG]: #21 src/pocketmine/PocketMine(274): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[39] C:\Users\~~~\Documents\Pocketmine-MP\, string[47] C:\Users\~~~\Documents\Pocketmine-MP\plugins\)
[19:01:46] [Server thread/DEBUG]: #22 (1): require(string[94] phar://C:/Users/astro/Documents/Pocketmine-MP/PocketMine-MP.phar/src/pocketmine/)
```

## Config
...
---
§6§lStarter Kit:
  # Format: "id:damage:count:name:ench_name:ench_level"

  # If you want only custom name (no enchantments): "id:damage:count:name"

  # If you don't want enchantments or custom name: "id:damage:count"

  # If you want only enchantments (no custom name): "id:damage:count:DEFAULT:ench_name:ench_level" -- you have to put DEFAULT in the name field

  # If you want more than one enchantment just do: "id:damage:count:name:ench1_name:ench1_level:ench2_name:ench2_level"
  # or "id:damage:count:DEFAULT:ench1_name:ench1_level:ench2_name:ench2_level" if you don't want a custom item name

  # Please note: You have to write numeric IDs
  items:
  - "272:0:1:DEFAULT"
  - "274:0:1:DEFAULT"
  - "275:0:1:DEFAULT"
  - "364:0:32"
  helmet: "302:0:1"
  chestplate: "303:0:1"
  leggings: "304:0:1"
  boots: "305:0:1"

  commands: # {player} will get replaced by player's name
  - "tell {player} §6§lStarter Kit §aclaimed!"

  cooldown:
    hours: 0
    minutes: 10

  # Format: "name:time:amplifier"
  # Time is in seconds
  
  #Kit name to show on the menu, you can use color symbol '§'
  kit-name: "§6§lStarter Kit"

  # Add a cost for the kit. Compatible with EconomyAPI only
  # Put 0 if you want the kit to be free
  money: 0
  
  #Enter image url for the kit. png image. The image will be shown in kit main menu if its compatible.
  image-url: "https://minecraft.gamepedia.com/media/minecraft.gamepedia.com/f/f2/Stone_Sword.png?version=ede6f1623f23eb3959b3d5287dc54bb0"

  # Info to show when selected a Kit in UI form
  info: "Starter Kit for those starting out"
  
  # Custom Kit perms, CaSe sEnSiTiVe
  permission: "kit.starter"
§l§2No§4DeBuff §9Kit:
  # Format: "id:damage:count:name:ench_name:ench_level"

  # If you want only custom name (no enchantments): "id:damage:count:name"

  # If you don't want enchantments or custom name: "id:damage:count"

  # If you want only enchantments (no custom name): "id:damage:count:DEFAULT:ench_name:ench_level" -- you have to put DEFAULT in the name field

  # If you want more than one enchantment just do: "id:damage:count:name:ench1_name:ench1_level:ench2_name:ench2_level"
  # or "id:damage:count:DEFAULT:ench1_name:ench1_level:ench2_name:ench2_level" if you don't want a custom item name

  # Please note: You have to write numeric IDs
  items:
  - '276:0:1:DEFAULT:17:3'
  - '368:0:16:DEFAULT'
  - '438:21:33:DEFAULT'
  - '364:0:64'
  helmet: "310:0:1:17:3"
  chestplate: "311:0:1:17:3"
  leggings: "312:0:1:17:3"
  boots: "313:0:1:17:3"

  commands: # {player} will get replaced by player's name
  - "tell {player} §l§2No§4DeBuff §9Kit §aclaimed!"

  cooldown:
    hours: 0
    minutes: 1

  # Format: "name:time:amplifier"
  # Time is in seconds
  effect: "speed:3600:2"
  #Kit name to show on the menu, you can use color symbol '§'
  kit-name: "§l§2No§4DeBuff §9Kit"

  # Add a cost for the kit. Compatible with EconomyAPI only
  # Put 0 if you want the kit to be free
  money: 0
  
  #Enter image url for the kit. png image. The image will be shown in kit main menu if its compatible.
  image-url: "https://www.digminecraft.com/brewing_recipes/images/splash_potion_healing.png"

  # Info to show when selected a Kit in UI form
  info: "NoDeBuff Kit for PVP"
  
  # Custom Kit perms, CaSe sEnSiTiVe
  permission: "kit.nodebuff"
...

**Comment by AL3XPI** [comment]
@Infernus101

**Comment by Infernus101** [comment]
@AL3XPI you're using old KitUI, try with latest release

**Comment by AL3XPI** [comment]
@Infernus101 I used the latest release and it still gives me the same error.

**Comment by Infernus101** [comment]
> @Infernus101 I used the latest release and it still gives me the same error.

Whats the version?

**Comment by AL3XPI** [comment]
1.3.5

**Comment by Infernus101** [comment]
> 1.3.5

do you have old piggyce in your plugins?

**Comment by AL3XPI** [comment]
i have piggy ce 1.4.6 latest version

**Comment by Infernus101** [comment]
send a screenshot of console

**Comment by AL3XPI** [comment]
![image](https://user-images.githubusercontent.com/35903333/61715370-fc9ead80-ad2a-11e9-93cc-8de1649a90e0.png)

**Comment by Infernus101** [comment]
do /plugins and send output

**Comment by AL3XPI** [comment]
![image](https://user-images.githubusercontent.com/35903333/61725784-6d9b9080-ad3e-11e9-8cda-f1f8d77375ee.png)
@Infernus101

**Comment by Infernus101** [comment]
@AL3XPI fixed in v1.3.6

**Comment by AL3XPI** [comment]
thanks it works now
